projects/python-client/src/datapane/client/api/ipython_utils.py

```diff
@@ -194,3 +204,17 @@ def cells_to_blocks(ipython_output_cache: dict, opt_out: bool = True) -> typing.
         display_msg("No blocks found.")
 
     return blocks
+
+
+@capture_event("Notebook to App")
```

Comment:
If we have the event here shall we remove from `cells_to_blocks` - I don't think that will be used much directly

Reply:
If we're not interested in capturing the individual usage, I'll remove this!